pkgs/servers/nosql/neo4j/default.nix

```diff
@@ -11,18 +11,24 @@ stdenv.mkDerivation rec {
     sha256 = "1zjb6cgk2lpzx6pq1cs5fh65in6b5ccpl1cgfiglgpjc948mnhzv";
   };
 
-  nativeBuildInputs = [ makeWrapper ];
+  nativeBuildInputs = [ makeWrapper bashInteractive ];
 
   installPhase = ''
     mkdir -p "$out/share/neo4j"
     cp -R * "$out/share/neo4j"
 
     mkdir -p "$out/bin"
+    compgen_wrapper="$out/share/neo4j/bin/compgen"
+    cat << _EOF_ > $compgen_wrapper
```

Comment:
it's really a nitpick, but I've never seen the file redirection after the EOF, is there a reason for this?
did you consider the following
```
cat > "$comgen_wrapper" << _EOF_
...
```

Reply:
Actually looking around, I've now seen some more occurences of this, feel free to ignore if you feel strongly about that choice.

Reply:
I'm used to having file redirects being at the end of the line and didn't even think about it. But think your way is clearer. Made the change.